docs/cudf/source/user_guide/io.md

```diff
@@ -1,10 +1,15 @@
-I/O Supported dtypes
-====================
+# Input / Output
 
-The following table lists are compatible cudf types for each supported IO format.
+This page contains Input / Output related APIs in cuDF.
 
-.. rst-class:: io-supported-types-table special-table
+## I/O Supported dtypes
+
+The following table lists are compatible cudf types for each supported
+IO format.
+
+```{eval-rst}
 .. table::
+    :class: io-supported-types-table special-table
```

Comment:
This table gets rendered incorrectly and I'm investigating:

<img width="1272" alt="Screen Shot 2022-04-26 at 7 38 09 AM" src="https://user-images.githubusercontent.com/3190405/165292363-12e34374-c44a-4dd5-8ae4-721c3977dade.png">